### ISSUE TITLE
Commands not registered

### ISSUE THREAD

**Issue by SebScoFr** [FriendsOfSymfony/FOSOAuthServerBundle#508]
I want to use the new `fos:oauth-server:create-client` command recently merged in `1.5`.
Bundle is correctly installed, though for some reason when I run `bin/console` I can see no command under the `fos` namespace, as if they are not registered by Symfony. I'm using SF4.

Is anyone else experiencing this?

**Comment by SebScoFr** [comment]
As a dirty workaround, it works if I add the following to my **config/services.yaml** file:
```yaml
services:
    ...
    FOS\OAuthServerBundle\Command\:
        resource: '../vendor/friendsofsymfony/oauth-server-bundle/Command'
```
But obviously I shouldn't have to do that

**Comment by dkarlovi** [comment]
@SebScoFr note that the bundle is not fully auto-configured by Flex, as discussed in #501

**Comment by SebScoFr** [comment]
True. Until that is the case then, I'll just add that workaround to my services.yaml whenever I need to run the command.

**Comment by dkarlovi** [comment]
@SebScoFr could you send a PR on top of 1.6 which fixes this?

**Comment by SebScoFr** [comment]
@dkarlovi I've just opened #534

**Comment by dkarlovi** [comment]
Fixed in #534, thanks again @SebScoFr
